.circleci/config.yml

```diff
@@ -793,12 +796,20 @@ jobs:
       - run:
           name: Halt job if already built code coverage today.
           command: |
-            NOW=`date +%Y-%M-%d`
+            NOW=`date +%Y-%m-%d`
             LAST=`cat /home/circleci/lastbuild` || true
             if [[ "$LAST" == "$NOW" ]]; then
               echo Last build occured today, halting.
               circleci step halt
+            else
+              echo Last build occured $LAST, building.
+              date +%Y-%m-%d > /home/circleci/lastbuild
             fi
+      - save_cache:
+          name: store updated daily latch file.
+          key: code-coverage-daily-{{ epoch }}
```

Comment:
notice the use of {{ epoch }}, required for the cache to get written.   Cache keys come in parts separated by dashes....  on a fetch if the prefix matches get the latest cache written.   Wonky but works.